easybuild/easyconfigs/w/waves2Foam/waves2Foam-20200703-foss-2019b.eb

```diff
@@ -0,0 +1,94 @@
+easyblock = 'Bundle'
+
+name = 'waves2Foam'
+# datestamp of this SVN revision (2140)
+version = '20200703'
+
+homepage = 'https://openfoamwiki.net/index.php/Contrib/waves2Foam'
+description = "The library waves2Foam is a toolbox used to generate and absorb free surface water waves."
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'pic': True}
+
+dependencies = [
+    ('OpenFOAM', 'v1912'),
+    ('GSL', '2.6'),
+]
+
+local_oceanwave3d_optflags = "-ffpe-trap=invalid,zero,overflow -ffree-line-length-none -fstack-protector-all"
+local_oceanwave3d_common_opts = 'USER=easybuild LINLIB="-lopenblas -L%(installdir)s/lib -lskit" FC="$FC" '
```

Comment:
What is the purpose of the `USER` variable? We do not have any user called `easybuild` in our system.